.github/workflows/run-tests.yml

```diff
@@ -6,18 +6,47 @@ name: Run Tests
 on: [push, pull_request]
 
 jobs:
+  call_run_tests-react-18:
+    uses: yext/slapshot-reusable-workflows/.github/workflows/run_tests.yml@v1
+    with:
+      build_script: |
+        npx playwright install
```

Comment:
do you know what change is making us have to install the playwright browsers now when we didn't need to before?

Reply:
not sure. does github workflow usually have it installed somewhere? if so, maybe a version got upgraded.

Reply:
github workflows don't install it automatically, so my guess is that one of the storybook-related package upgrades you made included some change related to this. can you track that down in their release notes to see if we can avoid this somehow?

Reply:
looks like  storybook@^7.4.5 and @storybook/test-runner@^0.15.2  that we are using for the react17 and 16 test runners are the same versions we have been using before this change. it may be difficult to figure out which dependency exactly is not installing playwright.

Reply:
using storybook@^7.4.5 is different from storybook@7.4.5. by pinning to the major, we're likely using the same version for react 16, 17, and 18 tests (v7.6.13) assuming that's the latest storybook version. you could try pinning to v7.4.5 by installing `storybook@7.4.5` and removing `npx playwright install` to see if it works.

is the storybook version bump needed at all? we're still pinned to the same minor so I'd be surprised if the change was significant enough to block react 18 support